content/knowledge-others/install-unity-version.md

```diff
@@ -36,19 +34,24 @@ Then you can continue building your Unity apps as descriped [here](../yaml-quick
 - `2021.3.12f1`
 - `2021.3.13f1`
 - `2021.3.15f1`
+{{< /tab >}}
+{{% tab header="2020.X" %}}
+- `2020.3.15f2`
+- `2020.3.21f1`
+- `2020.3.38f1`
+- `2020.3.40f1`
+- `2020.3.41f1`
+{{< /tab >}}
+{{< /tabpane >}}
 
-{{<notebox>}}
-These editiors are `x86_64` have only the `macOS`, `Android`, and `iOS` modules, if you need other modules then you need to install it using [Unity Hub CLI](./-others/install-unity-version/#unity-installation-script).
-{{</notebox>}}
-
-{{< /collapse >}}
-
+If you can't find your desired Unity version in the list, please contact us [here](https://codemagic.io/contact/).
 
 {{<notebox>}}
-If you can't find your desired Unity version in the list, please contact us [here](https://codemagic.io/contact/).
+**Notes:**
+- These versions are the `Unity Editor (macOS x86_64)`, and have only the `macOS`, `Android`, and `iOS` modules.
+- If your app require additional modules then you need to install it using [Unity Hub CLI](./-others/install-unity-version/#unity-installation-script) like this: `/Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub -- --headless install-modules --version <UNITY_VERSION> -m windows-mono`, this will install the windows modules for the spesfied `<UNITY_VERSION>`.
```

Comment:
```suggestion
- If your app requires additional modules then you need to install it using [Unity Hub CLI](./-others/install-unity-version/#unity-installation-script) like this: `/Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub -- --headless install-modules --version <UNITY_VERSION> -m windows-mono`, this will install the windows modules for the specified` <UNITY_VERSION>`.
```